data/json/items/tools.json

```diff
@@ -1162,6 +1162,26 @@
     "use_action": "WATER_PURIFIER",
     "flags": [ "ALLOWS_REMOTE_USE" ]
   },
+  {
+    "id": "lifestraw",
+    "type": "TOOL",
+    "name": "lifestraw",
+    "description": "Set the lifestraw in suspect water, let sit for one minute then drink. The two part filtration system will purify the water you drink.  Water taken from uncertain sources like a river may be dirty.",
+    "weight": "104 g",
+    "volume": "592 ml",
+    "price": 4000,
+    "price_postapoc": 75000,
+    "to_hit": -3,
+    "bashing": 1,
+    "material": [ "cotton", "plastic" ],
+    "symbol": "i",
```

Comment:
Could you add a look_like to this? It would help make #34230 a little easier.

Reply:
Would I add looks_like "pipe" or looks_like "lifestraw"?

Reply:
I can also add additional details to the description so that eventually if someone wants to build a specific tile for it it will be easier?

Reply:
For `looks_like`, you use the id of the thing you want it to look like.